apps/passporting-server/README.md

```diff
@@ -1,5 +1,133 @@
 # idOS Passporting Server
 
+## Overview
+idOS Passporting Service provides an endpoint for inserting delegated access grants into the idOS system. This service represents Obligated Entity 1 (OE1), which originally issued a credential to a user. Another entity, Obligated Entity 2 (OE2), can use this service to request access to that credential by creating an access grant.
+
+The service is built using the `Hono` web framework and validates incoming requests using the `Zod` validation library. It also utilizes cryptographic functions for secure key management and signature validation.
+
+## Features
+- Validates incoming requests with a structured schema.
+- Authenticates requests using bearer tokens.
+- Transmits Delegated Access Grant (DAG) data to idOS.
+- Ensures security with key-based cryptographic operations.
+
+## Endpoint Details
+### Base URL
+`http://localhost:3000`
+
+### Endpoints
```

Comment:
This feels overly verbose for what we're trying to provide. The server is a single file that's easy enough to read. If we ever evolve into something more complicated, the right way to document endpoints would be using an OpenAPI spec.

Removing this part for now.